react/features/variant/VariantForm.js

```diff
@@ -105,9 +110,7 @@ export const VariantForm = (props) => {
 		}
 	}
 
-	const hide_imputchip_radio = dtype != undefined ?
-		dtype == 'chip' ? true : false
-	: false
+	const hide_imputchip_radio = dtype != null ? dtype == 'chip' ? true : false : false
```

Comment:
Can this be just const hide_imputchip_radio = dtype == 'chip' ?

Reply:
Makes more sense,  thanks

Reply:
Thank you for the comments! Merged as suggested current master to the branch and resolved conflicts.